lib/json_api_resource/queryable.rb

```diff
@@ -25,6 +25,7 @@ def find(id)
       def create(attr = {})
         run_callbacks :create do
           new(:client => self.client_klass.create(attr))
+          self.meta = self.client.last_request_meta
```

Comment:
`create` can't run callbacks because it's not an instance method. It is never actually used because it will explode as currently implemented. It should be removed, or changed to not have the callbacks